<?php

namespace ContainerPFkdi9G;

use Symfony\Component\DependencyInjection\Argument\RewindableGenerator;
use Symfony\Component\DependencyInjection\Exception\RuntimeException;

/**
 * @internal This class has been auto-generated by the Symfony Dependency Injection Component.
 */
class get_ServiceLocator_Y4EzA8mService extends App_KernelDevDebugContainer
{
    /**
     * Gets the private '.service_locator.Y4EzA8m' shared service.
     *
     * @return \Symfony\Component\DependencyInjection\ServiceLocator
     */
    public static function do($container, $lazyLoad = true)
    {
        return $container->privates['.service_locator.Y4EzA8m'] = new \Symfony\Component\DependencyInjection\Argument\ServiceLocator($container->getService, [
            'passwordEncoder' => ['services', 'security.password_encoder', 'getSecurity_PasswordEncoderService', true],
            'security' => ['privates', 'security.helper', 'getSecurity_HelperService', true],
            'user' => ['privates', '.errored..service_locator.Y4EzA8m.App\\Entity\\User', NULL, 'Cannot autowire service ".service_locator.Y4EzA8m": it references class "App\\Entity\\User" but no such service exists.'],
            'validator' => ['services', '.container.private.validator', 'get_Container_Private_ValidatorService', false],
        ], [
            'passwordEncoder' => '?',
            'security' => '?',
            'user' => 'App\\Entity\\User',
            'validator' => '?',
        ]);
    }
}
